theano/sandbox/gpuarray/basic_ops.py

```diff
@@ -599,50 +681,53 @@ def c_code(self, node, name, inputs, outputs, sub):
     def c_code_cache_version(self):
         return (3,)
 
-
 cuda_from_gpu = CudaFromGpu()
 
 
 class GpuAlloc(HideC, Alloc):
-    def __init__(self, memset_0=False):
+    __props__ = ('memset_0', 'context') + Alloc.__props__
```

Comment:
Why the + alloc.__props__? We override the init and do not call Alloc.__init__, so we would loose all props that Alloc create. (but in practice, it alloc.props is empty, but I think it is a bad practice.